source/common/stats/allocator_impl.h

```diff
@@ -11,6 +11,7 @@
 
 #include "absl/container/flat_hash_set.h"
 #include "absl/strings/string_view.h"
+#include "metric_impl.h"
```

Comment:
`#include "source/common/stats/metric_impl.h"`

Reply:
Not sure how that got added. Done.